### PR TITLE
GLT-581 after delete, displays table from which item was deleted

### PR DESCRIPTION
**Pull request by callunity**

### CODE REVIEW

File: miso-web/src/main/webapp/scripts/instituteDefaults_ajax.js [hunk 1]
```diff
@@ -989,7 +994,12 @@ var Options = Options || {
   },
 
   confirmDelete: function (endpoint) {
+	var pieces = endpoint.split('/');
+	var singular = pieces[pieces.length - 2];
+	localStorage['tablediv'] = singular + 's';
     if (endpoint.indexOf("sampleclass") != -1) {
+      // need a modified tablediv because of pluralization
```
Comment (apmasell) [inline miso-web/src/main/webapp/scripts/instituteDefaults_ajax.js]: Yikes.

**Comment by apmasell** [comment]
:+1:

File: miso-web/src/main/webapp/scripts/instituteDefaults_ajax.js (hunk 1 above)
Comment (callunity) [inline miso-web/src/main/webapp/scripts/instituteDefaults_ajax.js]: You make a good point. I'll change the table div #ids to be the same as the singular resource which will avoid all pluralization messiness.

**Comment by tonydebat** [comment]
👍

**Comment by tonydebat** [comment]
Nice! The modified update/delete behaviour gets rid of all the problems with refreshing.

**Pull request merged**